config/ucx.conf

```diff
@@ -9,6 +9,9 @@ UCX_GDR_COPY_BW=0MBs,get_dedicated:30GBs,put_dedicated:30GBs
 UCX_GDR_COPY_LAT=200ns
 UCX_DISTANCE_BW=auto,sys:16500MBs
 
+UCX_CUDA_IPC_LAT=2.5e-6
```

Comment:
why need a space line before?

Reply:
It is not required but I decided to add it there to improve readability of config.